portal/src/graphql/portal/DesignScreen/DesignScreen.tsx

```diff
@@ -827,32 +829,33 @@ const Preview: React.VFC<PreviewProps> = function Preview(props) {
         return;
       }
       setSelectedPreviewPage(option.key as PreviewPage);
+      setisIframeLoading(true);
     },
     []
   );
 
   const src = useMemo(() => {
     const url = new URL(effectiveAppConfig.http?.public_origin ?? "");
     url.pathname = selectedPreviewPage;
-    url.searchParams.append("x_color_scheme", designForm.state.selectedTheme);
     url.searchParams.append("ui_locales", designForm.state.selectedLanguage);
     return url.toString();
   }, [
     effectiveAppConfig.http?.public_origin,
     designForm.state.selectedLanguage,
-    designForm.state.selectedTheme,
     selectedPreviewPage,
   ]);
 
   const onLoadIframe = useCallback(() => {
     const message = mapDesignFormStateToPreviewCustomisationMessage(
       designForm.state
     );
+    setisIframeLoading(false);
     authUIIframeRef.current?.contentWindow?.postMessage(message, "*");
   }, [designForm.state]);
 
   return (
     <div className={cn("flex", "flex-col", className)}>
+      {isIframeLoading ? <ShowLoading /> : null}
```

Comment:
Please pay more attention to the ui you introduce

<img width="1645" alt="Screenshot 2024-08-16 at 6 06 10 PM" src="https://github.com/user-attachments/assets/7a53b2f3-bd44-4eab-bd46-65f8d3b2402b">